projects/lib/src/board/board.cpp

```diff
@@ -471,6 +471,11 @@ bool Board::setFenString(const QString& fen)
 		m_squares[i] = Piece::WallPiece;
 	m_key = 0;
 
+	int maxsymlen = 1;
```

Comment:
Thank you. This is coming near a full circle. I already had this kind of solution in an earlier version, but then I did not calculate the necessary symbol length yet. I used a fixed length of one or two (per override in a protected method e.g. for Crazyhouse).  I will combine this according to your suggestion.